nym-vpn-app/src-tauri/src/sentry.rs

```diff
@@ -23,6 +44,17 @@ pub fn init(os: &OsInfo) -> Option<ClientInitGuard> {
             enable_logs: true,
             shutdown_timeout: Duration::from_secs(1),
             server_name: Some(Cow::Borrowed("nym")),
+            before_send: Some(Arc::new(|mut event| {
+                if let Some(message) = &event.message {
```

Comment:
Nit: perhaps matching against `warn` and `error` levels could save us from parsing lengthy trace/debug messages that we're not interested in. What do you think?